cve_bin_tool/output_engine/html.py

```diff
@@ -1,13 +1,34 @@
 import os
 from collections import Counter
+from collections import defaultdict
 from datetime import datetime
 from typing import Set
 
 import plotly.graph_objects as go
 from jinja2 import Environment, FileSystemLoader
 
+from .util import group_cve_by_remark
 from ..cve_scanner import CVEData
+from ..input_engine import Remarks
 from ..log import LOGGER
+from ..version import VERSION
+
+
+def render_cves(vendor, product, version, hid, cve_row, tag, cves):
+    list_cves = []
```

Comment:
Could we get a docstring here? (the comment below might be all you need, but it's always nice to have proper docstrings so things like fancy ides can use them.)